molecule/default/tasks/openshift_process.yml

```diff
@@ -162,3 +162,25 @@
 
 - assert:
     that: result is not changed
+
+# Processing template without message
+- name: create template with file {{ files_dir }}/pod-template.yaml
+  kubernetes.core.k8s:
```

Comment:
hmm. test is failing on this task with the message `Failed to retrieve requested object: You must provide a body when calling methods on a ResourceList`

Reply:
ok maybe it was working locally because I am not using ``kubernetes.core:1.2.0``